### DISCUSSION TITLE
Rewind and forward buttons

### DISCUSSION THREAD

**Discussion by belbernard** [videojs/video.js#8296] [Ideas]
For use in technical and/or foreign-language podcasts, we need rewind and forward buttons that allow you to skip approximately 10 seconds at a time. Is this envisaged on videojs?

**Comment by amtins** [comment]
@belbernard video.js since version [8.2.0](https://github.com/videojs/video.js/releases/tag/v8.2.0 ) provides skip buttons, the link to the documentation documentation https://videojs.com/guides/options/#skipbuttons

**Reply by gkatsev** [reply]
And there's https://github.com/mister-ben/videojs-seek-buttons if you need it for older versions of Video.js.

**Reply by belbernard** [reply]
Great, it works. Thank you!
